docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -9,141 +9,125 @@ description: "A dual-region setup allows you to run Camunda in two regions synch
+- The regions (for example, two Kubernetes clusters) must be able to communicate with each other (for example, via VPC peering). See [example implementation](/self-managed/setup/deploy/amazon/amazon-eks/dual-region.md) for AWS EKS.
   - Kubernetes services in one cluster must be resolvable and reachable from the other cluster and vice-versa. This is essential for proper communication and functionality across regions:
     - For AWS EKS setups, ensure DNS chaining is configured. Refer to the [Amazon Elastic Kubernetes Service (EKS) setup guide](/self-managed/setup/deploy/amazon/amazon-eks/dual-region.md).
     - For OpenShift, [Submariner](https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.11/html/networking/networking#submariner) is recommended for handling multi-cluster networking. Specific implementation guides are not yet available.
 - Maximum network round trip time (**RTT**) between regions should not exceed **100 ms**.
 - Required open ports between the two regions:
-  - **9200** for Elasticsearch (for cross-region data push by Zeebe).
+  - **9200** for Elasticsearch (for cross-region data pushed by Zeebe).
   - **26500** for communication to the Zeebe Gateway from clients/workers.
   - **26501** and **26502** for communication between Zeebe brokers and Zeebe Gateway.
 
 ### Zeebe Cluster Configuration
 
-Only a combination for Zeebe broker counts and replication factors is supported:
+The following Zeebe brokers and replication configuration is supported:
 
 - `clusterSize` must be a multiple of **2** and at least **4** to evenly distribute brokers across the two regions.
 - `replicationFactor` must be **4** to ensure even partition distribution across regions.
-- `partitionCount` is unrestricted but should be chosen based on workload requirements. See [understanding sizing and scalability behavior](../../../components/best-practices/architecture/sizing-your-environment.md#understanding-sizing-and-scalability-behavior).
-- For more details on partition distribution, see [documentation on partitions](../../../components/zeebe/technical-concepts/partitions.md).
-
-### Regional Failure Management
+- `partitionCount` is unrestricted but should be chosen based on workload requirements. See [understanding sizing and scalability behavior](../../../components/best-practices/architecture/sizing-your-environment.md#understanding-sizing-and-scalability-behavior). For more details on partition distribution, see [documentation on partitions](../../../components/zeebe/technical-concepts/partitions.md).
 
-Customers are responsible for detecting regional failures and executing the [operational procedure](./../../operational-guides/multi-region/dual-region-ops.md).
-
-### Camunda 8 dual-region limitations
+### Camunda 8 Dual-Region limitations
```

Comment:
maybe for the technical writers to have a look over.

I would have kept it at `dual-region` or maximum at `Dual-region` (looks odd to me mid sentence) but at least `Dual-Region` I was shunned for in the past.